### PR TITLE
TRIVIAL: RAIL-1868, BB-1703, TET-85 partial - Update translations

### PR DESCRIPTION
**Pull request by petrjanu**

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5435/ : SUCCESS in 8m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/932/ : SUCCESS in 6s

File: src/internal/translations/pt-BR.json [hunk 1]
```diff
@@ -79,7 +83,7 @@
     "properties.yaxis.title": "Eixo Y",
     "properties.axis.labels": "rótulos",
     "properties.axis.rotation": "Rotação",
-    "properties.axis.name": "nome",
+    "properties.axis.name": "nme",
```
Comment (no23reason) [inline src/internal/translations/pt-BR.json]: Probably an error – the pt-PT does not have this change.

Comment (petrjanu) [reply src/internal/translations/pt-BR.json]: Same error again. Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5442/ : SUCCESS in 9m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/940/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2820/ : SUCCESS in 15m 19s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5446/ : SUCCESS in 16m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1324/ : SUCCESS in 23s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1815/ : SUCCESS in 21m 41s

**Pull request merged**